core/src/main/java/org/apache/calcite/rel/core/TableFunctionScan.java

```diff
@@ -64,6 +64,7 @@
    *
    * @param cluster        Cluster that this relational expression belongs to
    * @param inputs         0 or more relational inputs
+   * @param traits         Traits
```

Comment:
Can you rename this parameter to `traitSet`. That is the preferred name these days. Thanks for adding javadoc.

Reply:
With pleasure.